src/main/java/org/fundacionjala/coding/miguel/SortInnerContent.java

```diff
@@ -0,0 +1,34 @@
+package org.fundacionjala.coding.miguel;
+
+public class SortInnerContent {
+    public static final int MAX_CHAR = 26;
+    public static final int LIMIT_WORDS = 4;
+
+    public String sortInnerContent(final String sentence) {
```

Comment:
This may re-written like this
```java
    public String sortInnerContent(final String sentence) {
        String[] words = sentence.split(" ");
        for (int i = 0; i < words.length; i++) {
            if (words[i].length() >= LIMIT_WORDS) {
                String[] body = words[i].substring(1, words[i].length() - 1).split("");
                Arrays.sort(body, Comparator.reverseOrder());
                words[i] = new StringBuilder(words[i])
                        .replace(1, words[i].length() - 1, String.join("", body)).toString();
            }
        }
        return String.join(" ", words);
    }
```